woocommerce/google-listings-and-ads
# Use `merge-trunk-develop-pr` action

### Changes proposed in this Pull Request:

Use `merge-trunk-develop-pr` action instead of `automerge-released-trunk`.
We need a manual PR merge due to our branch protection rules.

### Screenshots:

![image](https://github.com/woocommerce/grow/assets/17435/56dfeb95-fe43-43cf-8048-c6055b196879)

![image](https://github.com/woocommerce/grow/assets/17435/5cfc7184-c6de-41de-97b2-9f09dc0b023b)


### Detailed test instructions:



1. Merge this PR :see_no_evil: or fork repo with this branch merged
2. Create a release PR with `woocommerce/grow/prepare-extension-release` action
3. Merge the release PR
4. Check that a new PR to merge trunk to develop is created


### Additional details:


### Changelog entry

> Dev - Use `merge-trunk-develop-pr` action
